src/App.jsx

```diff
@@ -5,42 +5,42 @@ import NavBar from './components/NavBar/NavBar';
 export default function App() {
   return (
     <div>
-      <NavBar type='welcome' />
+      <NavBar type="welcome" />
```

Comment:
I think in terms of eslint someone already fixed it to a single quote, so please keep all single quoted!!

Reply:
How should I change my eslinter setting, because my linter doesn't like a single quote? After I fixed these into a single quote, my whole file went red.

Reply:
sorry i just noticed this!! my bad, eslint likes it double quoted...! so yea lets keep this double quoted.